src/util/is.ts

```diff
@@ -1,12 +1,14 @@
 import { Zone } from '../types';
 
+export const isString = (value: unknown): boolean => Object.prototype.toString.call(value) === '[object String]';
```

Comment:
At first you said we should remove this entire file. Now you're adding back functions. Have you changed your mind about it's usefulness?